lib/travis/build/script/nix.rb

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+# Maintained by
+#  - Domen Kožar
+#  - Rok Garbas
+#  - Matthew Bauer <mjbauer95@gmail.com>
```

Comment:
This information needs to be displayed in the logs. See https://github.com/travis-ci/travis-build/blob/1bfb57d836416e5fda9d5cdc8986d681bbc4a9d7/lib/travis/build/script/csharp.rb#L20-L21 for an example.